Update onboarding docs with ROSA Classic scenarios (#38)
CSPI-QE/ocp-ci-docs: docs/OCP_CI_Tutorials/Scenario_Development/Scenario_Development_Guide.md

        scenario mtr
```

### `firewatch-rosa-aws-sts` & `firewatch-rosa-aws-sts-hypershift` <!-- omit from toc -->

step-registry doc (ROSA Classic): [firewatch-rosa-aws-sts](https://steps.ci.openshift.org/workflow/firewatch-rosa-aws-sts)

step-registry doc (ROSA Hypershift): [firewatch-rosa-aws-sts-hypershift](https://steps.ci.openshift.org/workflow/firewatch-rosa-aws-sts-hypershift)

These workflows are going to serve as the base installation mechanism for our ROSA Classic/Hypershift OpenShift installation, 
and can be used in a very similar way to how we use ipi-aws.

> As you can see in the link above this workflow is equipped with steps for deprovisioning which include must-gather that we get for free simply just by using this. This also is complete with the [firewatch-report-issues ref](https://steps.ci.openshift.org/reference/firewatch-report-issues) which can be used to create Jira issues based on failures.

Reviewer: ```suggestion
These workflows includes the steps for deprovisioning including `must-gather`. It also includes the [firewatch-report-issues ref](https://steps.ci.openshift.org/reference/firewatch-report-issues) which can be used to create Jira issues based on failures.
```

    workflow: firewatch-rosa-aws-sts-hypershift
```

> - To access and the ROSA cli as our workflows does, we need to use the dynamic image tags of both rosa and ocm cli:

Reviewer: ```suggestion
- To access the ROSA cli in a similar way as the above workflows, we need to use the dynamic image tags of both rosa and ocm cli:
```
